app/assets/stylesheets/active_admin/structure/_main_structure.scss

```diff
@@ -1,29 +1,29 @@
-#wrapper {
-  width: 100%;
-}
-
-.index #wrapper {
-  display: table;
-}
-
 #active_admin_content {
-  margin: 0;
+  @include clearfix;
   padding: $horizontal-page-margin;
 
-  #main_content_wrapper {
-    float: left;
+  // so index table and filters scroll horizontally
+  &.with_sidebar {
+    display: table;
     width: 100%;
-
-    #main_content{
-      margin-right: 300px;
-    }
   }
+}
 
-  &.without_sidebar #main_content_wrapper #main_content{ margin-right: 0; }
+#main_content_wrapper {
+  float: left;
+  width: 100%;
+}
 
-  #sidebar {
-    float: left;
-    width: $sidebar-width;
-    margin-left: -$sidebar-width;
-  }
+#main_content {
```

Comment:
Avoid using id selectors<br>Selector `main_content` should be written in lowercase with hyphens